sites/fast-component-explorer/app/explorer.tsx

```diff
@@ -112,6 +113,9 @@ class Explorer extends Foundation<
 
     private viewerContainerRef: React.RefObject<HTMLDivElement> = React.createRef();
     private viewerContentAreaPadding: number = 20;
+    private maxViewerHeight: number = 0;
+    private maxViewerWidth: number = 0;
+    private windowResizing: NodeJS.Timeout;
```

Comment:
Where is `NodeJS.Timeout` coming from? We won't generally be executing in a node env, should this be something else?

Reply:
Weirdly, that's what window.timeout returns as its type, I don't really know why.

Reply:
Woop, it resolves if I use the `window.setTimout`.

Reply:
huh...